bin/spiced/src/lib.rs

```diff
@@ -226,7 +226,9 @@ pub async fn run(args: Args) -> Result<()> {
 
     let cloned_rt = rt.clone();
     let server_thread =
-        tokio::spawn(async move { cloned_rt.start_servers(args.runtime, tls_config).await });
+        tokio::spawn(
```

Comment:
Why did we have to pin this now?

Reply:
I was getting this lint error: https://rust-lang.github.io/rust-clippy/master/index.html#large_futures

Reply:
I just got it too 😆